assets/javascripts/discourse/templates/mobile/list/topic-list-item.raw.hbs

```diff
@@ -57,13 +67,20 @@
     {{/unless}}
   </div>
 
-  {{#if socialStyle}}
-    <div class='main-link-footer'>
-      {{raw "list/topic-meta" title=view.title likesHeat=view.likesHeat topic=topic}}
+  {{#if tilesOrSocial}}
```

Comment:
@merefield Again, see if you can simplify the conditional logic here by putting more of it in the component.

Reply:
I've reverted this file to the current HEAD. All layout changes are now done with jQuery in the Component. A little worried about performance now though.